Assignment/oddeven.txt

```diff
@@ -0,0 +1,21 @@
+/* [16] WAP to check number is Even or Odd
+ID : 23BCA261 */
+
+#include<iostream.h>
+#include<conio.h>
+void main()
+{
+	int n;
+	clrscr();
+	cout<<"Enter a number : ";
+	cin>>n;

```

Comment:
what is the use of cin?

Reply:
cin is used to take input from user. same like scanf.